LICENSE.txt

```diff
@@ -1,28 +1,4 @@
 Copyright (c) 2008-2011, Kenneth Bell
-
-Permission is hereby granted, free of charge, to any person obtaining a
-copy of this software and associated documentation files (the "Software"),
-to deal in the Software without restriction, including without limitation
-the rights to use, copy, modify, merge, publish, distribute, sublicense,
-and/or sell copies of the Software, and to permit persons to whom the
-Software is furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in
-all copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
-FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
-DEALINGS IN THE SOFTWARE.
-
---
-
-Certain files:
-
-
```

Comment:
Wouldn't it be best to have the LICENSE file always, but also use the license file expression for nuget purposes?

The code is available without nuget..

Reply:
The GitHub diff doesn't show it very nicely, but the MIT licenses text is included _twice_ in the `LICENSE.txt` file. This change removes the _first_ copy of the MIT license, and keeps the second.

This is what the file looks like after the changes: https://github.com/DiscUtils/DiscUtils/blob/b86548309d17f909705807b548c489419f7cb8c4/LICENSE.txt

I think/hope that after this change, GitHub is able to recognize this as the MIT license and tag the repository as being MIT-licensed.

You can check via https://api.github.com/repos/discutils/discutils/license

Reply:
Ooh.. You're right. We lose the "certain files" part, but that's probably fine.